### ISSUE TITLE
Chinese Charset "特色" Can Cause Crash

### ISSUE THREAD

**Issue by qqzj** [chrisboulton/php-diff#28]
Hello,I'm from China.
When I use this php-diff,I was surprised! It's awsome!
but when I diff tow file like follow:
a.txt
构建具有中国<b>特色</b>的医学人才培养体系
b.txt
构建具有中国的医学人才培养体系
The php-diff crashed!

**Comment by JBlond** [comment]
I added unicode support in JBlond/php-diff

**Comment by JBlond** [comment]
See https://github.com/JBlond/php-diff

**Comment by Bilge** [comment]
Yes but you deleted the Composer file so nobody can actually use it.

**Comment by JBlond** [comment]
Why is composer needed to use it? Clone it, download the zip file.

**Comment by Bilge** [comment]
🤦‍♂️

**Comment by JBlond** [comment]
I've added my own composer file now
